one-pager-maker/src/pages/Edit.tsx

```diff
@@ -12,10 +11,14 @@ import {documentApi} from "../api/documentApi.ts";
 import {viewHistoryApi} from "@/api/viewHistoryApi.ts";
 
 function Edit() {
-  const uid = useAppSelector(state => state.user.user?.uid);
-  const params = useParams<{ id: string }>();
+  const {uid, documentId} = useParams<{ uid: string, documentId: string }>();
 
-  const document = documentApi.useGetDocumentQuery({ uid: uid ?? "", documentId: params.id ?? ''});
+  if (uid === undefined || documentId === undefined) {
+    // this is called when route setting is wrong
+    return <main>Route setting is wrong</main>
```

Comment:
why is there a main tag?

Reply:
I typically use main tag for each page so that html tags looks like this:
```html
<header>...</header>
<main>...</main>
<footer>...</footer>
```

Reply:
Thank you!